lib/mix/tasks/oban_web.install.ex

```diff
@@ -0,0 +1,142 @@
+defmodule Mix.Tasks.ObanWeb.Install.Docs do
```

Comment:
The Web modules are actually `Oban.Web`, though the package name is `oban_web`. I think it would be preferable to use `Mix.Tasks.Oban.Web.Install`, with the dot separation.

Reply:
When installing a package, Igniter looks for a task called `<project_name>.install`. We can look into supporting other install task names if you'd like.

Reply:
No need, that makes sense as a default name for the installer.